test/integration/noderesourcetopology_test.go

```diff
@@ -607,7 +624,234 @@ func TestTopologyMatchPlugin(t *testing.T) {
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "6G"},
+			},
+			statusErr: "cannot align container: cnt-3",
+		},
+		{
+			description: "[24][tier1] multi init containers with good allocation, multi-containers with over memory allocation - not fit",
+			initCntReq: []map[string]string{
+				{cpu: "30", memory: "10G"},
+				{cpu: "30", memory: "10G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "40G"},
+				{cpu: "2", memory: "60G"},
+			},
+			statusErr: "cannot align container: cnt-3",
+		},
+		{
+			description: "[27][tier1] multi init containers with good allocation, container with cpu over allocation - not fit",
+			initCntReq: []map[string]string{
+				{cpu: "30", memory: "10G"},
+				{cpu: "30", memory: "10G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "35", memory: "40G"},
+			},
+			statusErr: "cannot align container: cnt-1",
+		},
+		{
+			description: "[28][tier1] multi init containers with good allocation, multi-containers with good allocation - fit",
+			initCntReq: []map[string]string{
+				{cpu: "30", memory: "10G"},
+				{cpu: "30", memory: "10G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "40G"},
+			},
+		},
+		{
+			description: "[29][tier1] multi init containers when sum of their cpus requests (together) is over allocatable, multi-containers with good allocation - fit",
+			initCntReq: []map[string]string{
+				{cpu: "30", memory: "10G"},
+				{cpu: "30", memory: "10G"},
+				{cpu: "30", memory: "10G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "40G"},
+				{cpu: "2", memory: "6G"},
+			},
+		},
+		{
+			description: "[29][tier1] multi init containers when sum of their memory requests (together) is over allocatable, multi-containers with good allocation - fit",
+			initCntReq: []map[string]string{
+				{cpu: "3", memory: "50G"},
+				{cpu: "3", memory: "50G"},
+				{cpu: "3", memory: "50G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "20", memory: "40G"},
+				{cpu: "2", memory: "6G"},
+			},
+		},
+		{
+			description: "[32][tier1] multi init containers with over cpu allocation - not fit",
+			initCntReq: []map[string]string{
+				{cpu: "40", memory: "50G"},
+				{cpu: "3", memory: "50G"},
+				{cpu: "3", memory: "50G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "2", memory: "6G"},
+			},
+			statusErr: "cannot align init container: cnt-1",
+		},
+		{
+			description: "[32][tier1] multi init containers with over memory allocation - not fit",
+			initCntReq: []map[string]string{
+				{cpu: "20", memory: "50G"},
+				{cpu: "40", memory: "50G"},
+				{cpu: "3", memory: "50G"},
+			},
+			cntReq: []map[string]string{
+				{cpu: "20", memory: "40G"},
+				{cpu: "2", memory: "6G"},
```

Comment:
Also noticed the use of "G" in `60G` instead of `60Gi` here and in all the entries of `scopeEqualsContainerTests`.

G means 5 Gigabytes while Gi means Gibibytes. , For example:
5 G = 5000000 KB / 5000 MB
5 Gi = 5368709.12 KB / 5368.70 MB

I would prefer the use of units comparable to the ones we have in NRT object. If this is intentional however, please make it very explicit because this is very easy to miss.

Reply:
No. Actually, it should be the init container which is failed. I fixed that.
About the ```G``` notation, I used that because it was easier to calculate the values needed for the test.
But if the NRT objects are using ```Gi``` it won't make any difference.
I'll change that to ```Gi``` for consistency.